packages/repositories.config

```diff
@@ -1,5 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
+  <repository path="../src/NUnitFramework/framework/packages.config" />
+  <repository path="../src/NUnitFramework/nunitlite.tests/packages.config" />
+  <repository path="../src/NUnitFramework/nunitlite/packages.config" />
+  <repository path="../src/NUnitFramework/testdata/packages.config" />
+  <repository path="../src/NUnitFramework/tests/packages.config" />
```

Comment:
Was this required for a build? I thought `repositories.config` was a historical file, which was no longer used. We removed it in the console